ArduPlane/quadplane.cpp

```diff
@@ -483,7 +483,7 @@ bool QuadPlane::setup(void)
     }
     
     if (hal.util->available_memory() <
-        4096 + sizeof(*motors) + sizeof(*attitude_control) + sizeof(*pos_control) + sizeof(*wp_nav)) {
```

Comment:
Why don't we just malloc, check the result then look at the amount of free memory?